PAQ: Use inline images again, and some other changes

kailol-sag opened the pull request.

BeateRixen commented on content/streaming-analytics/analytics-builder-bundle/getting-started.md:
```diff
@@ -48,83 +48,59 @@ If the navigator is currently hidden, click the small arrow at the very left of
-2.  In the toolbar of the model editor, click the following icon to leave the model editor and thus to return to the model manager:
-    ![Close icon](/images/streaming-analytics/analytics-builder/icon-exit.png)
+1. In the toolbar of the model editor, click <img src="/images/streaming-analytics/analytics-builder/icon-save.png" alt="Save icon" style="display:inline-block; margin:0"> to save your newly created model.
+2. In the toolbar of the model editor, click <img src="/images/streaming-analytics/analytics-builder/icon-exit.png" alt="Close icon" style="display:inline-block; margin:0"> to leave the model editor and thus to return to the model manager
```
> click the close icon

BeateRixen commented on content/streaming-analytics/analytics-builder-bundle/getting-started.md:
```diff
@@ -181,33 +151,23 @@ The following is a brief description of the blocks that are defined within the s
+2. Click the actions menu of the **On missing measurements create alarm** sample and then click **Create model from sample**.
+   The new model is immediately shown in the model editor. It has the same name, description and tags as the sample.
+3. If you want to rename the model, click the model name which is shown at the left of the toolbar. You can then specify a new name in the resulting **Model Configuration** dialog box.
+4. In the toolbar of the model editor, click <img src="/images/streaming-analytics/analytics-builder/icon-save.png" alt="Save icon" style="display:inline-block; margin:0"> to save the new model.
```
See above

BeateRixen commented on content/streaming-analytics/analytics-builder-bundle/using-the-instance-editor.md:
```diff
@@ -28,8 +28,7 @@ Each row that is shown for an instance has an actions menu \(the three vertical
 
-If the following error icon is shown near the end of a row, the corresponding instance is no longer processing events. Click that icon to get more information.
-![Error icon](/images/streaming-analytics/analytics-builder/instance-editor-error.png)
+If the <img src="/images/streaming-analytics/analytics-builder/instance-editor-error.png" alt="Error icon" style="display:inline-block; margin:0"> error icon is shown near the end of a row, the corresponding instance is no longer processing events. Click that icon to get more information.
```
Please have the icon name (here "error icon") being followed by the image

BeateRixen commented on content/streaming-analytics/analytics-builder-bundle/using-the-instance-editor.md:
```diff
@@ -73,15 +71,12 @@ Once an instance is active, you cannot modify the template parameter values or m
-
-    In the case of an error, the following error icon is shown at the right of the table and the instance cannot be activated. Click the error icon to get more information.
-    ![Error icon](/images/streaming-analytics/analytics-builder/instance-editor-error.png)
+    In the case of an error, the <img src="/images/streaming-analytics/analytics-builder/instance-editor-error.png" alt="Error icon" style="display:inline-block; margin:0"> error icon is shown at the right of the table and the instance cannot be activated. Click the error icon to get more information.
```
See above

BeateRixen commented:
@kailol-sag @MWindel I started a review and added comments but we should first align on some general rules.

1. Icon names are not capitalized or formatted in a special way, so it is "save icon, close icon,". At least that is the way we use it right now.
2. Inline images for icons follow the icon name (plus "icon"), i.e. "save icon + image"
3. We should not use an icon without naming it, otherwise the reading flow gets interrupted. Exception could perhaps be the "X icon" as this simply shows an "X".

I have added several comments but stopped my review then to first align with you.

What do you think?

kailol-sag commented:
> @kailol-sag @MWindel I started a review and added comments but we should first align on some general rules.
> 
> 1. Icon names are not capitalized or formatted in a special way, so it is "save icon, close icon,". At least that is the way we use it right now.
> 2. Inline images for icons follow the icon name, i.e. "save icon + image"
> 3. We should not use an icon without naming it, otherwise the reading flow gets interrupted. Exception could perhaps be the "X icon" as this simply shows an "X".
> 
> I have added several comments but stopped my review then to first align with you.
> 
> What do you think?

Ok, @BeateRixen , I can change the wording for the inline icons as you suggested. 

But I'm confused now: I looked into several other places of the doc and only found icon names in the running text, but no image. Is this a new convention that I'm not aware of? However, I find the images helpful and would therefore prefer to keep them.

MWindel commented on content/streaming-analytics/analytics-builder-bundle/using-the-model-manager.md:
```diff
@@ -119,22 +99,18 @@ You can also create a new model from a sample. See [Creating a model from a samp
 You can edit \(or view\) each model that is currently listed in the model manager.
 
-When a model is active, editing will set the model to read-only mode. In this case, the model editor only allows you to view the contents of the model \(for example, you can view the block parameters\). You can navigate and zoom the model as usual, but you cannot change anything. The save button in the model editor is therefore disabled.
+When a model is active, editing will set the model to read-only mode. In this case, the model editor only allows you to view the contents of the model \(for example, you can view the block parameters\). You can navigate and zoom the model as usual, but you cannot change anything. The save icon in the model editor is therefore disabled.
```
@kailol-sag I would show the icon here. What do you think?

kailol-sag replied:
ok, I'll insert it.
